sync/views.py

```diff
@@ -28,18 +32,20 @@ def sync(request):
         # This _could_ cause timeouts, but might be OK?
         # Will depend on how many line items we get.
         for knack_value in knack_values:
-            if getattr(settings, 'DEBUG'):
-                print('sent {} to knack'.format(json.dumps(knack_value, indent=4)))
+            log_debug('sent {} to knack'.format(json.dumps(knack_value, indent=4)))
 
             return_status, result_string = knackload.load(json.dumps(knack_value))
 
             if return_status != 200:
-                print('Error: We failed to send {} to knack'.format(knack_value))
+                contribution_refcode = knack_value['field_638']
```

Comment:
'field_638' is the date. Please log the order_id from ActBlue instead.

Reply:
Also, this code will crash in debug mode because those fields won't exist in knack_value.